examples/aerial_liquidity_pool.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+import base64
+
+from cosmpy.aerial.client import LedgerClient, NetworkConfig
+from cosmpy.aerial.contract import LedgerContract
+from cosmpy.aerial.tx_helpers import SubmittedTx
+from cosmpy.aerial.wallet import LocalWallet
+from cosmpy.crypto.keypairs import PrivateKey
+
+
+def _wait_for_tx(operation: str, tx: SubmittedTx):
+    print(f"Waiting for {operation} to complete... (tx: {tx.tx_hash})")
+    tx.wait_to_complete()
+    print(f"Waiting for {operation} to complete... done \n")
+
+
+def main():
+
+    # Choose any key with tokens available
+    key = PrivateKey(
+        b"@\xa95\x9a'\x972\xb88\x8b\x89\xccw0\xba\xed\x99m\xe9\x8f\xf0\xcf]\xb4\x93\xb13\xb9I\xdc\x96u"
+    )
+    wallet = LocalWallet(key)
+
+    # Network configuration
+    cfg = NetworkConfig(
+        chain_id="dorado-1",
+        url="grpc+https://grpc-dorado.fetch.ai:443",
+        fee_minimum_gas_price=1000000000000,
+        fee_denomination="atestfet",
+        staking_denomination="atestfet",
+    )
+
+    ledger = LedgerClient(cfg)
+
+    # Choose a path to any local file
+    PATH = "any/path/to/a/local/file.any"
```

Comment:
Can we delete this and just use an empty string or `None` below?

Reply:
I see the problem. Just need a temporary solution here until https://github.com/fetchai/cosmpy/pull/163 is merged.

I tested with `Path = __file__` and that seems to work.

Reply:
I see the problem. Just need a temporary solution here until https://github.com/fetchai/cosmpy/pull/163 is merged.

I tested with `Path = __file__` and that seems to work.

Reply:
No, it has to be a path to any file, if not an error is produced. I opened PR 163 to have the option to include or not the PATH but Ed said that we should keep it there "we do really want to have the contract path so that we don't end up creating lots of duplicate contracts on the chain."

Reply:
Ok sounds good James